Updating README with latest build instructions

cppwfs opened the pull request:
* Removing old commands
* Requires readme in the UI directory for doc generation and one in the main dir for github.

resolves #436

sabbyanandan commented:
Apparently, we need the README in both places to aid with doc generation process.

cppwfs commented:
Updated. Squashed.

On Mon, Sep 11, 2017 at 2:37 PM, Sabby Anandan <notifications@github.com>
wrote:

> Apparently, we need the README in both places to aid with doc generation
> process.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/spring-cloud/spring-cloud-dataflow-ui/pull/437#issuecomment-328619709>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAqdeSF9vT7Uk17E9L9F7icmoInr8c94ks5shX4EgaJpZM4PTiAa>
> .
>
